### ISSUE TITLE
window dependency regression

### ISSUE THREAD

**Issue by josh** [6to5/6to5#628]
Hey @sebmck, hit an issue trying to run 3.0.12 in ruby-6to5.

It looks like the `debug` package added as a dependency in efaf56c6de048a62edd9c54bc46d27cf6377de1c has a dependency on a `window` global.

``` js
if (typeof chrome !== 'undefined' && typeof chrome.storage !== 'undefined')
  storage = chrome.storage.local;
else
  storage = window.localStorage;
```

In ExecJS's runtime, theres no `window` property.

```
ReferenceError: window is not defined
    at Object../debug (ruby-6to5/tmp/6to5-source-3.0.12/lib/6to5.js:25494:13)
```

I think they *may* have fixed this in newer versions of debug. It looks like now they do a feature check.

I'm wondering if theres anyway way to add a sanity check of the compiled `lib/6to5.js` to the CI processor here?

Thanks!!!

**Comment by sebmck** [comment]
Ah, crap. Looks like they have a `browser` property that uses [this](https://github.com/visionmedia/debug/blob/master/browser.js) file instead. *Should* be fixed by changing the require to `debug/node`.

> I'm wondering if theres anyway way to add a sanity check of the compiled lib/6to5.js to the CI processor here?

Hm, I can't think of anything at the moment but I'm open to suggestions.

**Comment by sebmck** [comment]
Actually, should be able to do something similar to what's done [here](https://github.com/6to5/6to5/blob/master/test/browserify.js).

**Comment by sebmck** [comment]
I've pushed out 3.0.15 that uses `debug/node` instead of `debug`. Let me know if it fixed it.

**Comment by josh** [comment]
:metal: thanks for the quick fix

**Comment by sebmck** [comment]
No problem! I commented too early, should be available on npm in a couple of minutes.

**Comment by josh** [comment]
Yep, that did the trick. Published 3.0.15 on the ruby end.

**Comment by josh** [comment]
Thanks again.
